source/change-streams.rst

```diff
@@ -339,19 +421,54 @@ Single Server Topologies
 
 Presently, change streams cannot be initiated on single server topologies as they do not have an oplog.  Drivers MUST NOT throw an exception in this scenario, but instead rely on an error returned from the server.  This allows for the server to seamlessly introduce support for this in the future, without need to make changes in driver code.
 
+startAtClusterTime
+^^^^^^^^^^^^^^^^^^
+
+:since: 4.0
+
+``startAtClusterTime`` specifies that a ``changeStream`` will only return changes that occurred after the specified ``Timestamp``.
+
+The server expects ``startAtClusterTime`` as a bson document of the form
+
+.. code:: typescript
+
+    {
+      ts: Timestamp;
+    }
+
+Drivers MUST allow users to specify a ``startAtClusterTime`` option in the ``Collection.watch`` and ``Database.watch`` helpers. They SHOULD allow users to specify this value as a raw ``Timestamp``, and SHOULD NOT force users to wrap the timestamp as the server specifies.
```

Comment:
I'd prefer MUST regarding the wrapping the timestamp.  Why even consider allowing this inconsistency?

Reply:
done